docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md

```diff
@@ -281,7 +281,7 @@ When to use:
 
 What to watch out for:
 
-- If you must use this because you wish to use `Async.Start` or another function that requires `Async<unit>`, consider if discarding the result is okay to do. Discarding results just to fit a type signature should not generally be done.
+- If you must use `Async.Ignore` because you wish to use `Async.Start` or another function that requires `Async<unit>`, consider if discarding the result is okay. Generally, you should not discard results just to fit a type signature.
```

Comment:
```suggestion
- If you must use `Async.Ignore` because you wish to use `Async.Start` or another function that requires `Async<unit>`, consider if discarding the result is okay. Avoid discarding results just to fit a type signature.
```